org.pitivi.Pitivi.json

```diff
@@ -350,12 +351,12 @@
         },
         "shared-modules/linux-audio/lv2.json",
         "shared-modules/linux-audio/lilv.json",
+        "shared-modules/libsoup/libsoup-2.4.json",
         {
             "name": "gstreamer",
             "buildsystem": "meson",
             "builddir": true,
             "config-opts": [
-                "-Dpygi-overrides-dir=/app/lib/python3.9/site-packages/gi/overrides/",
```

Comment:
In the dev sandbox we have `"-Dgst-python:pygi-overrides-dir=/app/lib/python3.10/site-packages/gi/overrides/",` shall we add it also here  @thiblahute ?

Reply:
it triggered a build error to have that option.

Reply:
I mean the one I removed. (it was set to python3.10)